keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlResourceRepository.kt

```diff
@@ -185,6 +188,7 @@ class SqlResourceRepository(
 
     private val UID = field<String>("uid")
     private val API_VERSION = field<String>("api_version")
+    private val SERVICE_ACCOUNT = field<String>("service_account")
```

Comment:
Does this need its own column in the database? Will we ever be indexing on this?

Reply:
Removing!